src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7621,6 +7621,13 @@ public sealed partial class StringBuilder : System.Runtime.Serialization.ISerial
         void System.Runtime.Serialization.ISerializable.GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public override string ToString() { throw null; }
         public string ToString(int startIndex, int length) { throw null; }
+        public ChunkEnumerator EnumerateChunks() { throw null; }
+        public struct ChunkEnumerator
+        {
+            public ChunkEnumerator GetEnumerator() { throw null; }
+            public bool MoveNext() { throw null;  }
+            public ReadOnlyMemory<char> Current { get { throw null; } }
```

Comment:
nit: reorder this to be in alphabetical order.

@weshaggard, we should also regenerate this ref to clean up some stuff.

For instance, the type isn't fully qualified here: https://github.com/dotnet/corefx/blob/master/src/System.Runtime/ref/System.Runtime.cs#L3743
And we are using abbreviated default syntax:
https://github.com/dotnet/corefx/blob/master/src/System.Runtime/ref/System.Runtime.cs#L5088
Lastly, overridden methods don't tend to have method bodies:
https://github.com/dotnet/corefx/blob/master/src/System.Runtime/ref/System.Runtime.cs#L5086

Reply:
Vance you normalize this by running our GenerateReferenceSource target on the ref (https://github.com/dotnet/corefx/blob/master/dir.targets#L120). Just run `msbuild /t:GenerateReferenceSource` in the ref folder (note you will need to undo some changes to the file for the changes that are in the System.Runtime.manual.cs).

Reply:
That will also remove the enumerator APIs that are getting added in this PR. I think we need to account for System.Private.CoreLib.dll as well (maybe we need to build corefx using the local build of coreclr by passing the path to msbuild /p:CoreCLROverridePath)?
https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md#testing-with-private-coreclr-bits

Can you please share the steps to do that as well?

Reply:
If the new APIs are added manually and you just round-trip the system.runtime reference assembly they will not be lost and it will normalize the ref.